meerevm/meer/genesis_check.go

```diff
@@ -2,15 +2,14 @@ package meer
 
 import (
 	"encoding/hex"
-
 	"github.com/Qitmeer/qng/params"
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-const MainAllocHash = "204bb2f79e29453543b74b620533c854ddd327603fdcadb5c938b4e29e2cbd0b"
-const MixAllocHash = "7630112cc9bbc47f2370d67890d40c98a7bf69d80d09f4c1c156a5ad373d46c6"
-const TestAllocHash = "7630112cc9bbc47f2370d67890d40c98a7bf69d80d09f4c1c156a5ad373d46c6"
-const PrivAllocHash = "7630112cc9bbc47f2370d67890d40c98a7bf69d80d09f4c1c156a5ad373d46c6"
+const MainAllocHash = "e36181b59aaec0602dbec5e16570e60ca3828abe07bf79fd08d3ca379cdae425"
```

Comment:
We need using the actual meerevm genesis hash for the test instead of the alloc hash.